framework/auth/__init__.py

```diff
@@ -48,6 +49,7 @@ def logout():
             del session.data[key]
         except KeyError:
             pass
+    Session.remove(Q('_id', 'eq', session._id))
```

Comment:
I don't think it will cause a problem, but if we're tracking any information outside the auth credentials on the Session object, those will be destroyed when the user logs out.

In the past we were keeping a history list of the user's visited pages there, but we no longer do that. I'm not aware of anything else that gets stored there, but that's why I asked :)